week-4/C-more-complex-objects/exercise-2.js

```diff
@@ -0,0 +1,42 @@
+/*
+    Given the same "house" object again
+
+    Follow the instructions below and then run the file 
+    and make sure it outputs the correct results
+*/
+
+var house = {
+    address: '1 Kinning Park',
+    previousOwners: ["Claire M.", "John A."],
+    currentOwner: {
+      firstName: 'Margaret',
+      lastName: 'Conway'
+    }
+};
+
+var newCurrentOwner = {
+    firstName: 'Margaret',
+    lastName: 'Conway'
+};

```

Comment:
Should this be a different name from `currentOwner`?

Reply:
Ah yea, it definitely should be :)) Good spot!